### PR TITLE
onboard esi-project

### PR DESCRIPTION
**Pull request by 4n4nd**
Resolves https://github.com/operate-first/support/issues/272

### CODE REVIEW

File: cluster-scope/base/core/namespaces/esi-operate-first-project/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
```
Comment (tumido) [inline cluster-scope/base/core/namespaces/esi-operate-first-project/kustomization.yaml:3]: missing `namespace` here, please apply https://github.com/operate-first/opfcli/pull/40

**Review by tumido — DISMISSED** [review]

**Comment by Gregory-Pereira** [comment]
Looks good.
/approve

**Review by tumido — COMMENTED** [review]
/lgtm

File: cluster-scope/base/core/namespaces/esi-operate-first-project/namespace.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: esi-operate-first-project
+    annotations:
+        openshift.io/description: Our project permits multiple tenants to flexibly allocate baremetal machines from a pool of available hardware, create networks, attach baremetal nodes and networks, and to optionally provision an operating system on those systems through the use of an associated provisioning service.
```
Comment (tumido) [inline cluster-scope/base/core/namespaces/esi-operate-first-project/namespace.yaml:6]: Actually, can we add a `display-name` here?

Comment (4n4nd) [reply cluster-scope/base/core/namespaces/esi-operate-first-project/namespace.yaml]: renamed to `display-name`

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Review by tumido — APPROVED** [review]
/lgtm
/approve
/unhold

**Pull request merged**